### PR TITLE
refactor: documentation (developer guides-agent setup)

### PR DESCRIPTION
**Pull request by bhavanakarwade**
### What?
- Refactored document for agent setup section

### CODE REVIEW

**Review by pranalidhanavade — APPROVED** [review]
LG

File: src/content/docs/en/contributorsguide/setup/agent/envsetup.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+SERVER_URL=     # Specify server url
+
+windowMs=    # Specify windowMs
+
+maxRateLimit=    # Specify maxRateLimit
+
+DID_CONTRACT_ADDRESS=    # Specify Did contract address
+
+# Specify Bcovrin test genesis
+**BCOVRIN_TEST_GENESIS=** `{"reqSignature":{},"txn":{"data":{"data":{"alias":"Node1","blskey":"4N8aUNHSgjQVgkpm8nhNEfDf6txHznoYREg9kirmJrkivgL4oSEimFF6nsQ6M41QvhM2Z33nves5vfSn9n1UwNFJBYtWVnHYMATn76vLuL3zU88KyeAYcHfsih3He6UHcXDxcaecHVz6jhCYz1P2UZn2bDVruL5wXpehgBfBaLKm3Ba","blskey_pop":"RahHYiCvoNCtPTrVtP7nMC5eTYrsUA8WjXbdhNc8debh1agE9bGiJxWBXYNFbnJXoXhWFMvyqhqhRoq737YQemH5ik9oL7R4NTTCz2LEZhkgLJzB3QRQqJyBNyv7acbdHrAT8nQ9UkLbaVL9NBpnWXBTw4LEMePaSHEw66RzPNdAX1","client_ip":"138.197.138.255","client_port":9702,"node_ip":"138.197.138.255","node_port":9701,"services":["VALIDATOR"]},"dest":"Gw6pDLhcBcoQesN72qfotTgFa7cbuqZpkX3Xo6pLhPhv"},"metadata":{"from":"Th7MpTaRZVRYnPiabds81Y"},"type":"0"},"txnMetadata":{"seqNo":1,"txnId":"fea82e10e894419fe2bea7d96296a6d46f50f93f9eeda954ec461b2ed2950b62"},"ver":"1"}
```
Comment (GHkrishna) [inline src/content/docs/en/contributorsguide/setup/agent/envsetup.md:42]:
```suggestion
BCOVRIN_TEST_GENESIS= `{"reqSignature":{},"txn":{"data":{"data":{"alias":"Node1","blskey":"4N8aUNHSgjQVgkpm8nhNEfDf6txHznoYREg9kirmJrkivgL4oSEimFF6nsQ6M41QvhM2Z33nves5vfSn9n1UwNFJBYtWVnHYMATn76vLuL3zU88KyeAYcHfsih3He6UHcXDxcaecHVz6jhCYz1P2UZn2bDVruL5wXpehgBfBaLKm3Ba","blskey_pop":"RahHYiCvoNCtPTrVtP7nMC5eTYrsUA8WjXbdhNc8debh1agE9bGiJxWBXYNFbnJXoXhWFMvyqhqhRoq737YQemH5ik9oL7R4NTTCz2LEZhkgLJzB3QRQqJyBNyv7acbdHrAT8nQ9UkLbaVL9NBpnWXBTw4LEMePaSHEw66RzPNdAX1","client_ip":"138.197.138.255","client_port":9702,"node_ip":"138.197.138.255","node_port":9701,"services":["VALIDATOR"]},"dest":"Gw6pDLhcBcoQesN72qfotTgFa7cbuqZpkX3Xo6pLhPhv"},"metadata":{"from":"Th7MpTaRZVRYnPiabds81Y"},"type":"0"},"txnMetadata":{"seqNo":1,"txnId":"fea82e10e894419fe2bea7d96296a6d46f50f93f9eeda954ec461b2ed2950b62"},"ver":"1"}
```

File: src/content/docs/en/contributorsguide/setup/agent/references.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: Reference
+description: Reference
+---
+
+- To know more about <a href = "https://aca-py.org/latest/">AFJ Controller</a>
```
Comment (GHkrishna) [inline src/content/docs/en/contributorsguide/setup/agent/references.md:6]:
```suggestion
- To know more about <a href = "https://aca-py.org/latest/">ACA-py</a>
```

**Review by GHkrishna — CHANGES_REQUESTED** [review]

File: src/content/docs/en/contributorsguide/setup/agent/envsetup.md (hunk 1 above)
Comment (bhavanakarwade) [reply src/content/docs/en/contributorsguide/setup/agent/envsetup.md]: Resolved

File: src/content/docs/en/contributorsguide/setup/agent/references.md (hunk 2 above)
Comment (bhavanakarwade) [reply src/content/docs/en/contributorsguide/setup/agent/references.md]: Resolved

**Review by GHkrishna — APPROVED** [review]

**Review by KulkarniShashank — APPROVED** [review]

**Pull request merged**